src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download \"%s\"</string>
+  <string name="error_syncing_format">Could not sync \"%s\"</string>
+  <string name="error_validating_format">Could not validate \"%s\"</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">%s downloaded</string>
```

Comment:
The problem with %s placed before any words can be confusing. It's a notification text, which is often single line. And %s is an application name, which can be long enough. This is why I placed %s after the text.

Reply:
How is it confusing?

Reply:
Imagine an "Application With Very Very Very Long Name". And the notification which contains only 1 line of text. Instead of "Finished downloading Application With..." it will display "Application With Very Very Long..."

Reply:
```suggestion
  <string name="finished_downloading_format">⭳ %s</string>
```
More often "Finished downloading" will be something super long, and "%s" will be out of bounds(?)

Reply:
Is it possible to use "Downloaded %s" or "Finished %s" then? Seems strange to me.
Actually, it's possible to get rid of this string completely and just place the full application name in the title without any "downloaded" or anything else.
I also don't like "⭳ %s" (just don't like icons in texts).